### ISSUE TITLE
Ability to decouple images

### ISSUE THREAD

**Issue by JohnONeill** [orthes/medium-editor-insert-plugin#391]
If multiple images are uploaded at one time (i.e. using one file dialog), they're "grouped" together under one div rather than appearing as separate uploads. Is there a way to prevent this from happening?

**Comment by orthes** [comment]
No, right now it's not possible to separate them. If you need this, feel free to create a pull request.
